client/src/pages/index.js

```diff
@@ -1,16 +1,18 @@
-import React, { Fragment } from 'react';
-import { Router } from '@reach/router';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 /** importing our pages */
 import Tracks from './tracks';
 import Track from './track';
 import Module from './module';
 
 export default function Pages() {
   return (
-    <Router primary={false} component={Fragment}>
-      <Tracks path="/" />
-      <Track path="/track/:trackId" />
-      <Module path="/track/:trackId/module/:moduleId" />
-    </Router>
+    <BrowserRouter>
+      <Routes>
+        <Route element={<Tracks />} path="/" />
+        <Route element={<Track />} path="/track/:trackId" />
+        <Route element={<Module />} path="/track/:trackId/module/:moduleId" />
```

Comment:
should this Module route even be here? Don't we invite them to create the whole module page in lift-off-lab? 🤔